Loop/Managers/LoopDataManager.swift

```diff
@@ -821,7 +762,7 @@ extension LoopDataManager {
         if insulinEffect == nil {
             self.logger.debug("Recomputing insulin effects")
             updateGroup.enter()
-            doseStore.getGlucoseEffects(start: nextEffectDate) { (result) -> Void in
+            doseStore.getGlucoseEffects(start: nextEffectDate, end: nil, basalDosingEnd: now()) { (result) -> Void in
```

Comment:
Also, in relation to `now()`, I think you are going to need to look for any default arguments that use `= Date()` throughout.